src/sidebar/services/test/autosave-test.js

```diff
@@ -0,0 +1,94 @@
+import * as annotationFixtures from '../../test/annotation-fixtures';
+import createFakeStore from '../../test/fake-redux-store';
+import { waitFor } from '../../../test-util/wait';
+
+import autosaveService from '../autosave';
+import { $imports } from '../autosave';
```

Comment:
These two lines can be merged into one `import autosaveService, { $imports } from ...`. The only reason some existing files don't do that is because it was easier to generate separate lines during the ES module conversion.